README.md

```diff
@@ -0,0 +1,25 @@
+# ethereum-cache-creator
+## How to use program in the current iteration
+* download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
```

Comment:
```suggestion
* Download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
```

There should be another link the might give you the most recent artifact, but I  am not sure about this.